### PR TITLE
fix NSImageAlignment enumeration values

### PR DESCRIPTION
**Pull request by SamSchott**
Fixed enumeration values for NSImageAlignment in `toga_cocoa.lib`.

## PR Checklist:

- [x] All new features have been tested
- [ ] All new features have been documented
- [x] I have read the **CONTRIBUTING.md** file
- [x] I will abide by the code of conduct

### CODE REVIEW

**Comment by SamSchott** [comment]
Ok, done.

File: src/cocoa/toga_cocoa/libs/appkit.py [hunk 1]
```diff
@@ -303,15 +303,18 @@ def NSColorUsingColorName(background_color):
...
+class NSImageAlignment(Enum):
+    Center = 0
+    Top = 1
+    TopLeft = 2
+    TopRight = 3
+    Left = 4
+    Bottom = 5
+    BottomLeft = 6
+    BottomRight = 7
+    Right = 8
```
Comment (freakboy3742) [inline src/cocoa/toga_cocoa/libs/appkit.py]: These numbers look like they've got out of alignment with the original constants...

Comment (SamSchott) [reply src/cocoa/toga_cocoa/libs/appkit.py]: How so?

Comment (freakboy3742) [reply src/cocoa/toga_cocoa/libs/appkit.py]: /me looks sheepish... for some reason, I thought this patch was *adding* some constants; but the whole point is to fix the values. Problem exists between my keyboard and chair :-)

Comment (SamSchott) [reply src/cocoa/toga_cocoa/libs/appkit.py]: No worries, that happens to all of us :)

**Pull request merged**